src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/DisputeController.java

```diff
@@ -263,6 +263,26 @@ public ResponseEntity<Void> verifyDisputeEmail(@PathVariable(name="id") @Paramet
 		return ResponseEntity.ok().build();
 	}
 
+	@Operation(summary = "Updates the email address of a particular Dispute.")
+	@ApiResponses({
+		@ApiResponse(responseCode = "200", description = "Ok. Email unverified."),
+		@ApiResponse(responseCode = "404", description = "Dispute with specified id not found"),
+		@ApiResponse(responseCode = "500", description = "Internal server error occured.")
+	})
+	@PutMapping("/dispute/{id}/email/reset")
+	public ResponseEntity<Void> resetDisputeEmail(
+			@PathVariable(name="id")
+			@Parameter(description = "The id of the Dispute to update.")
+			Long id,
+			@RequestParam(name="email", required=true)
```

Comment:
I remember from our discussion before that disputant will be able to provide blank email to remove previously provided email address and opt out using email. So in this case should we make this parameter required=false?

Reply:
Ohhhh, right. Yes will remove that required=false.